chapters/VK_EXT_present_timing/queries.adoc

```diff
@@ -0,0 +1,481 @@
+    pname:presentStageTarget must: not zero
+****
+
+include::{generated}/validity/structs/VkSwapchainPresentTimingCreateInfoEXT.adoc[]
+--
+
+[open,refpage='vkGetSwapchainTimingPropertiesEXT',desc='Obtain the display timing properties of the PE\'s display',type='protos']
+--
+
+To query the presentation engine's timing properties for a given swapchain,
+call:
+
+include::{generated}/api/protos/vkGetSwapchainTimingPropertiesEXT.adoc[]
+
+  * pname:device is the device associated with pname:swapchain.
+  * pname:swapchain is the swapchain to obtain timing properties for.
+  * pname:pSwapchainTimingProperties is a pointer to an instance of the
+    sname:VkSwapchainTimingPropertiesEXT structure.
+
+.Valid Usage
+****
+  * pname:swapchain must: have been created with a
+    slink:VkSwapchainPresentTimingCreateInfoEXT specified in the
+    slink:VkSwapchainCreateInfoKHR::pname:pNext chain.
+****
+
+include::{generated}/validity/protos/vkGetSwapchainTimingPropertiesEXT.adoc[]
+--
+
+[open,refpage='VkSwapchainTimingPropertiesEXT',desc='Structure containing the RC duration of a display',type='structs']
+--
+
+The sname:VkSwapchainTimingPropertiesEXT structure is defined as:
+
+include::{generated}/api/structs/VkSwapchainTimingPropertiesEXT.adoc[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * pname:refreshDuration is zero; or is an indication of the duration
+    of a refresh cycle.
+    If the presentation engine is operating as an FRR display, this is the
+    number of nanoseconds from the start of one refresh cycle to the
+    start of the next refresh cycle.
+    If the presentation engine is operating as an VRR display
+    (i.e. refresh cycles may: have variable length), this is the
+    minimum number of nanoseconds from the start of one refresh cycle
+    to the start of the next refresh cycle.
+  * pname:variableRefresh is undefined: if pname:refreshDuration is
+    zero; otherwise it is ename:VK_FALSE if the presentation engine is
+    operating as a FRR display, or ename:VK_TRUE if the presentation
+    engine is operating as a VRR display.
+
+Some platforms may: not provide timing properties until after at least one
+image has been presented to the pname:swapchain. If timing properties
+change for the pname:swapchain, these same platforms may: not provide
+updated results until after at least one additional image has been
+presented to the pname:swapchain.
+
+include::{generated}/validity/structs/VkSwapchainTimingPropertiesEXT.adoc[]
+--
+
+[NOTE]
+.Note
+====
+The rate at which an application renders and presents new images is known as
+the image present rate (IPR, a.k.a. frame rate).
+The inverse of IPR, or the duration between each image present, is the image
+present duration (IPD).
+In order to provide a smooth, stutter-free animation, an application needs
+its IPD to be a multiple of pname:refreshDuration.
```

Comment:
Add in a clause that this is for FRR.  For example,

"... smooth, stutter-free animation on an FRR display, an application needs ...."